x-pack/plugins/actions/server/sub_action_framework/register.ts

```diff
@@ -54,5 +54,6 @@ export const register = <Config extends ActionTypeConfig, Secrets extends Action
     supportedFeatureIds: connector.supportedFeatureIds,
     validate: validators,
     executor,
+    renderParameterTemplates: connector.renderParameterTemplates,
```

Comment:
Seems like we should have a test for this new parameter being copied in, in `register.test.ts`.